services/src/unit-test/java/uk/gov/moj/sdt/services/TargetApplicationSubmissionServiceTest.java

```diff
@@ -655,35 +722,34 @@ public void processDlqRequestRejected() {
         individualRequest.setErrorLog(errorLog);
         individualRequest.setRequestStatus(IIndividualRequest.IndividualRequestStatus.REJECTED.getStatus());
 
-        this.mockIndividualRequestDao.persist(individualRequest);
-        EasyMock.expectLastCall();
-
         final IBulkSubmission bulkSubmission = individualRequest.getBulkSubmission();
 
-        EasyMock.expect(
-                this.mockIndividualRequestDao.queryAsCount(EasyMock.same(IndividualRequest.class), EasyMock.isA(Supplier.class))).andReturn(0L);
+        when(
+                this.mockIndividualRequestDao.queryAsCount(eq(IndividualRequest.class), isA(Supplier.class))).thenReturn(0L);
 
         mockIndividualRequestDao.persist(bulkSubmission);
-        EasyMock.expectLastCall();
+        verify(mockIndividualRequestDao).persist(bulkSubmission);
 
-        EasyMock.replay(mockIndividualRequestDao);
-        EasyMock.replay(mockCacheable);
-        EasyMock.replay(mockErrorMsgCacheable);
+        ListAppender<ILoggingEvent> listAppender = new ListAppender<>();
+        listAppender.start();
+        logger.addAppender(listAppender);
 
         this.targetAppSubmissionService.processDLQRequest(individualRequest, requestStatus);
 
-        EasyMock.verify(mockIndividualRequestDao);
-        EasyMock.verify(mockCacheable);
-        EasyMock.verify(mockErrorMsgCacheable);
+        List<ILoggingEvent> logList = listAppender.list;
 
-        Assert.assertEquals("Bulk submission status is incorrect", IBulkSubmission.BulkRequestStatus.COMPLETED
-                .getStatus(), individualRequest.getBulkSubmission().getSubmissionStatus());
-        Assert.assertNotNull("Bulk submission completed date should be populated", individualRequest
-                .getBulkSubmission().getCompletedDate());
-        Assert.assertNotNull("Bulk submission updated date should be populated", individualRequest
-                .getBulkSubmission().getUpdatedDate());
-        Assert.assertEquals("Individual Request should not be marked as dead letter", false,
-                individualRequest.isDeadLetter());
+        assertFalse(logList.isEmpty());
```

Comment:
If the test expects specific messages to be written to the log it should check for them rather than just checking it contains any messages.

Reply:
Test is still just checking if any messages have been written to the log rather than any specific messages. Might be worth checking the test to see if there's any need to check the log in this test - if not then it might be worth removing the log checks.

Reply:
done